README.md

```diff
@@ -81,7 +81,7 @@ dolphinscheduler-dist/target/apache-dolphinscheduler-incubating-${latest.release
 ### Thanks
 
 Dolphin Scheduler is based on a lot of excellent open-source projects, such as google guava, guice, grpc, netty, ali bonecp, quartz, and many open-source projects of Apache and so on.
-We would like to express our deep gratitude to all the open-source projects which contribute to making the dream of Dolphin Scheduler comes true. We hope that we are not only the beneficiaries of open-source, but also give back to the community. Besides, we expect the partners who have the same passion and conviction to open-source will join in and contribute to the open-source community!
+We would like to express our deep gratitude to all the open-source projects assist Dolphin Scheduler. We hope that we are not only the beneficiaries of open-source, but also give back to the community. Besides, we expect the partners who have the same passion and conviction to open-source could join in and contribute to the open-source community!
```

Comment:
`We would like to express our deep gratitude to all the open-source projects assist Dolphin Scheduler` should be `We would like to express our deep gratitude to all the open-source projects used in Dolphin Scheduler`

Reply:
`we expect the partners who have the same passion and conviction to open-source could join in and contribute to the open-source community!` should be `we hope that partners who have the same enthusiasm and passion for open source could join in and contribute to the open-source community!`

Reply:
I update the relate content, thanks for all your greate work